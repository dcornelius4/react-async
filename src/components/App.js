import React, { useState } from 'react';
import CharacterQuotes from '../containers/CharacterQuotes';

export default function App() {
  const [quotes, updateQuotes] = useState('character');

  return (
    <>
    <button onClick={() => updateQuotes('character')}>Character Quotes</button>
    {quotes === 'character' && <CharacterQuotes />}
    </>
  );
}
