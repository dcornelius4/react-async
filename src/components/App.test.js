import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Component', () => {
  it('render an app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
