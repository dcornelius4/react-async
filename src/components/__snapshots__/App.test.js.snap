// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component render an app 1`] = `<AllCharacters />`;
